java/src/main/java/com/google/appengine/tools/mapreduce/DatastoreMutationPool.java

```diff
@@ -62,7 +63,8 @@ private static final RetryerBuilder getRetryerBuilder() {
       .retryIfExceptionOfType(ApiProxyException.class)
       .retryIfExceptionOfType(ConcurrentModificationException.class)
       .retryIfExceptionOfType(CommittedButStillApplyingException.class)
-      .retryIfExceptionOfType(DatastoreTimeoutException.class);
+      .retryIfExceptionOfType(DatastoreTimeoutException.class)
+      .withStopStrategy(StopStrategies.stopAfterAttempt(8));
```

Comment:
this missed stop strategy too

Reply:
arguably this can have a bigger number of attempts

Reply:
i'm fine with this; are you adding it because you think I missed this in conversion from previous retryer implementation? or just because you think it's a good idea.

Reply:
In this particular case I added bc I think is OK to stop at some time and not try indefinitely

Reply:
OK ... but before they had several cases where it retried indefinitely. In some places, I intentionally omitted stop strategies to maintain the same behavior.

My question is if you were changing this code because you think I did that incorrectly, or just because you disagree with how they implemented it before. 

Either way, I'm certainly not bothered by capping it at some high number.